tests/natural_language/LanguageToolBearTest.py

```diff
@@ -39,6 +39,20 @@
     settings={'languagetool_disable_rules': 'UPPERCASE_SENTENCE_START'})
 
 
+LanguageToolBearDeprecatedSettingTest1 = verify_local_bear(
+    LanguageToolBear,
+    valid_files=('A correct English sentence sounds nice to everyone.',),
+    invalid_files=('Ein korrekter englischer Satz klingt für alle gut.',),
+    settings={'language': 'en-US'})
+
+
+LanguageToolBearDeprecatedSettingTest1 = verify_local_bear(
+    LanguageToolBear,
+    valid_files=('A correct English sentence sounds nice to everyone.',),
+    invalid_files=('Ein korrekter englischer Satz klingt für alle gut.',),
+    settings={'locale': 'en-US'})
+
```

Comment:
:+1: , thanks for adding these tests, we can be sure that the other deprecated settings work